### DISCUSSION TITLE
How can I use JobListener in Remote Client Example?

### DISCUSSION THREAD

**Discussion by Hiwen** [quartznet/quartznet#562] [General] [answered]
### Expected behavior

I want to add a JobListener in a Remote Scheduler.

### Actual behavior

But I got a error message when I run the program, which says : "Operation not supported for remote schedulers".

Is JobListener not support in Romote Mode in Quartz.net v2.6.1?
And will it supported in newest version?
Or will it be support in the future?

Thanks very much!

### Steps to reproduce

```csharp
// scheduler and job configuration, SystemTime prefereably set to fixed point
```

**Comment by lahma** [comment, marked as answer]
You need to add the listener when creating the scheduler (the one that listens to remote commands). You cannot add new listeners via remoting, need to be locally.

**Comment by Hiwen** [comment]
Thanks for Response!

Can I get Job Result through Any Other Way in Remote Mode ?

@lahma

**Comment by lahma** [comment]
You can record job history to for example database and query that.
